docs/background/glossary.md

```diff
@@ -58,10 +60,12 @@ Please note that this Glossary is **work in progress**. Do you encounter missing
 * frame (reading frame): **frame 1** is the normal reading frame, using the first nucleotide of each coding triplet of the annotated amino acid reference sequence for translation, starting at the A of the ATG translation initiation codon (nucleotide c.1): **frame 2** is the reading frame using the second nucleotide of the annotated amino acid reference sequence as first nucleotide of a coding triplet for translation in the shifted reading frame: **frame 3** is the reading frame using the third nucleotide of the annotated amino acid reference sequence as first nucleotide of a coding triplet for translation in the shifted reading frame.
 
 * frame shift: a sequence change between the translation initiation (start) and termination (stop) codon where, compared to a reference sequence, translation shifts to another reading frame ([_protein_](../../recommendations/protein/frameshift/))
-    
-* fusion transcript: a transcript (RNA molecule) which consist of parts of transcripts from two or more genes, resulting from a translocation, deletion, or inversion.
 
-* haplotype: contiguous set of genetic variants that are co-located on one chromosome (molecule) and are inherited from the same parent
+* gene fusion: the joining of two or more genes resulting in a chimeric transcript and/or a novel interaction between a rearranged regulatory element with the expressed product of a partner gene (a regulatory fusion).
```

Comment:
per @jtdendunnen:
> regarding "gene fusion" I assume this term is used on DNA level, correct?  If so I suggest; the joining, at the DNA level, of two or more genes....

Reply:
Not exactly; “gene fusion” is a complex concept that involves both genes (DNA) and their products (RNA).